app/models/chess_board.rb

```diff
@@ -1,49 +1,77 @@
 class ChessBoard
-
-  attr_accessor :board, :squares
+  include GamesHelper
+  attr_accessor :board
 
   def initialize
-    @ranks = (1..8).to_a
-    @files = ('A'..'H').to_a
-    @squares = create_squares
     @board = new_board
   end
 
-  def create_squares
-    squares = []
-    8.times do |i|
-      row = []
-      8.times { |j| row << Square.new([i, j]) }
-      squares << row
-    end
-    squares
-  end
-
   def new_board
-    squares[0][0].piece = Rook.new(false, 0, 0)
-    squares[0][7].piece = Rook.new(false, 0, 7)
-    squares[0][1].piece = Knight.new(false, 0, 1)
-    squares[0][6].piece = Knight.new(false, 0, 6)
-    squares[0][2].piece = Bishop.new(false, 0, 2)
-    squares[0][5].piece = Bishop.new(false, 0, 5)
-    squares[0][3].piece = Queen.new(false, 0, 3)
-    squares[0][4].piece = King.new(false, 0, 4)
-    squares[7][0].piece = Rook.new(true, 7, 0)
-    squares[7][7].piece = Rook.new(true, 7, 7)
-    squares[7][1].piece = Knight.new(true, 7, 1)
-    squares[7][6].piece = Knight.new(true, 7, 6)
-    squares[7][2].piece = Bishop.new(true, 7, 2)
-    squares[7][5].piece = Bishop.new(true, 7, 5)
-    squares[7][3].piece = Queen.new(true, 7, 3)
-    squares[7][4].piece = King.new(true, 7, 4)
-    squares[1].each_with_index.map do |sq, col|
-      sq.piece = Pawn.new(false, 1, col)
+    board =[]
+    8.times do
+      board <<  Array.new(8)
```

Comment:
Unnecessary spacing detected.<br>Operator `<<` should be surrounded by a single space.